app/controllers/concerns/paginatable.rb

```diff
@@ -0,0 +1,35 @@
+module Paginatable
```

Comment:
I'll likely be changing the name of this as part of the other work to remove AMS